homeassistant/components/media_player/__init__.py

```diff
@@ -433,7 +431,17 @@ async def async_get_media_image(self):
         if url is None:
             return None, None
 
-        return await _async_fetch_image(self.hass, url)
+        return await self._async_fetch_image_from_cache(url)
+
+    async def async_get_browse_image(
+        self, media_content_type, media_content_id, media_image_id=None
+    ):
```

Comment:
It would be good to type this method so the return value types become clear.

Reply:
i have some followup tests and such so ill update with such

Reply:
added here #43076